server/form-pages/apply/add-documents/attachDocuments.ts

```diff
@@ -68,23 +72,28 @@ export default class AttachDocuments implements TasklistPage {
         'N/A': 'No documents attached',
       }
     }
-
     this.body.selectedDocuments.forEach(d => {
       response[d.fileName] = d.description || 'No description'
     })
-
     return response
   }
 
   errors() {
     const errors: TaskListErrors<this> = {}
-
-    this.body.selectedDocuments.forEach(document => {
-      if (!document.description) {
-        errors[`selectedDocuments_${document.id}`] =
-          `You must enter a description for the document '${document.fileName}'`
-      }
-    })
+    // Check number of docs selected is docs is OK.
```

Comment:
Code is quite self-explanatory here, this comment doesn't feel needed.

Reply:
True